client/my-sites/themes/single-site-wpcom.jsx

```diff
@@ -21,6 +22,7 @@ export default connectOptions(
 
 		return (
 			<ThemeShowcase { ...props } siteId={ siteId }>
```

Comment:
I think the order shouldn't matter too much, thanks to the way React works -- as soon as `QuerySitePlans` is rendered, it will fetch data from the network and update Redux state. Which will in turn cause a re-render of state-`connect()`ed components that use plans data.